.vsconfig

```diff
@@ -17,7 +17,6 @@
     "Microsoft.VisualStudio.Component.AppInsights.Tools",
     "Microsoft.Net.Component.4.8.TargetingPack",
     "Microsoft.VisualStudio.Component.DiagnosticTools",
-    "Microsoft.NetCore.Component.Runtime.6.0",

```

Comment:
do we need to add one for 8.0?

Reply:
It's actually already in the list (above), it's just that the list isn't sorted.